Formula/circleci.rb

```diff
@@ -15,9 +16,11 @@ class Circleci < Formula
   end
 
   depends_on "go" => :build
+  depends_on "packr" => :build
 
   def install
     ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "on"
```

Comment:
I think if get rid of the use of `GOPATH` we shouldn't need to set this.

Reply:
I tried getting rid of `GOPATH`, but as I mentioned in my original comment, I just couldn't get it to work. The patch I've included (and even without using `GO111MODULE`) causes this error:
```
panic: stat /usr/local/Homebrew/Library/Taps/homebrew/_data/data.yml: no such file or directory

goroutine 1 [running]:
github.com/CircleCI-Public/circleci-cli/cmd.MakeCommands(0xc00009c058)
	/private/tmp/circleci-20210611-14083-1cwltoc/cmd/root.go:89 +0xfdc
github.com/CircleCI-Public/circleci-cli/cmd.Execute()
	/private/tmp/circleci-20210611-14083-1cwltoc/cmd/root.go:36 +0x25
main.main()
	/private/tmp/circleci-20210611-14083-1cwltoc/main.go:9 +0x25
```
This is without using `trimpath`, which is incompatible with `packr`.

Here's the `install` method, if it helps:
```ruby
def install
  system "packr2", "build", "-v"

  ldflags = %W[
    -s -w
    -X github.com/CircleCI-Public/circleci-cli/version.packageManager=homebrew
    -X github.com/CircleCI-Public/circleci-cli/version.Version=#{version}
    -X github.com/CircleCI-Public/circleci-cli/version.Commit=#{Utils.git_short_head}
  ]
  system "go", "build", "-ldflags", ldflags.join(" "), "-o", bin/"circleci"
end
```

Reply:
Ah, missed that. That's weird.

Reply:
Strangely, using the same steps in the debug mode of `brew install -sd circleci` (I made the build fail on purpose to enter the debug mode) results in a binary that seems to be fine 😅.

Reply:
My statement about `-trimpath` being incompatible, which was based on a comment `concourse/concourse#4342`, seems to have been incorrect – that was for `packr` v1.